cluster-provision/k8s/1.19/provision.sh

```diff
@@ -2,12 +2,22 @@
 
 set -ex
 
-function docker_pull_retry() {
+KUBEVIRTCI_SHARED_DIR=/var/lib/kubevirtci
+mkdir -p $KUBEVIRTCI_SHARED_DIR
+cat << EOF > $KUBEVIRTCI_SHARED_DIR/common.sh
+#!/bin/bash
+set -ex
+KUBELET_CGROUP_ARGS="--cgroup-driver=systemd --runtime-cgroups=/systemd/system.slice --kubelet-cgroups=/systemd/system.slice"
```

Comment:
maybe worth to add EXPORT ?
(it will still work without it since it source it, but in case other script is ran within the script and use those, it wont see those)
unless you prefer the scope to be explicit that its not exported.

Reply:
Done